src/pudl_zenodo_storage/zs/metadata.py

```diff
@@ -13,18 +12,33 @@
 following resources:
 <ul>
   <li><a href="https://github.com/catalyst-cooperative/pudl">The PUDL Repository on GitHub</a></li>
-  <li><a href="https://readthedocs.org/projects/catalystcoop-pudl/">PUDL Documentation</a></li>
+  <li><a href="https://catalystcoop-pudl.readthedocs.io">PUDL Documentation</a></li>
```

Comment:
I saw this link was wrong (it went to the RTD backend, not the docs) so I fixed it.